### PR TITLE
Test user platform on event tests

### PR DESCRIPTION
**Pull request by emmawillis**

### CODE REVIEW

File: harness/helpers.ts [hunk 1]
```diff
@@ -430,3 +430,7 @@ export const expectErrorMessageToBe = (message: string, expected: string) => {
     // the exception and the expected exception message
     expect(message).toBeTruthy()
 }
+
+export const getPlatformBySdkName = (name: string) => {
+    return name === 'DotNet' ? 'C#' : name // TODO use Sdks.dotnet instead of 'DotNet' when enable dotnet
```
Comment (ajwootto) [inline harness/helpers.ts:435]: I think DotNet has a distinction between Cloud and Local for the platform..

Comment (emmawillis) [reply harness/helpers.ts:435]: oh wait you're right, it shows up as just C# in snowflake but the actual event has cloud/local as well

Comment (ajwootto) [reply harness/helpers.ts:435]: hmm how come it shows up that way in Snowflake then?

Comment (emmawillis) [reply harness/helpers.ts:435]: not sure let me check again

Comment (emmawillis) [reply harness/helpers.ts:435]: ah never mind the events i was looking at were from May, i'm assuming before we added local as an option - the recent ones have the right format

**Review by ajwootto — APPROVED** [review]

**Pull request merged**